ArticleTemplates/assets/scss/modules/_premium-taster.scss

```diff
@@ -0,0 +1,67 @@
+.ad-free-taster {
+    margin: 12px 0;
+    display: inline-block;
+    border-radius: 50%;
+    background: color(brand400);
+    color: white;
+    line-height: 1;
+    box-sizing: content-box;
+
+    width: calc(100vw - 24px);
+    height: calc(100vw - 24px);
```

Comment:
How does this look on tablet?

Reply:
We are not running the test on tablets. phones only.